### DISCUSSION TITLE
CORS policy

### DISCUSSION THREAD

**Discussion by gravit22** [emilk/egui#344] [Q&A] [answered]
When I try to upload my app on web server, I got this error `has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.`
How to resolve it?

**Comment by lunabunn** [comment]
Fix your web server settings?

**Reply by gravit22** [reply]
How?

**Reply by lunabunn** [reply]
Depends on your web server setup, obviously.

**Reply by gravit22** [reply]
I do not understand. I uploaded files of template project on free web hosting. I have access to .htaccess file, and I tryed to add there 
`<IfModule mod_headers.c>
   Header set Access-Control-Allow-Origin "*"
 </IfModule>`
 But it didn't work, and I am asking for help.

**Comment by gravit22** [comment]
Resolved by adding appropriate headers and registering SSL. 
But there is another error:
`TypeError: WebAssembly.instantiate(): Import #0 module="__wbindgen_placeholder__" error: module is not an object or function`
https://learn-ithkuil.epizy.com/

**Comment by gravit22** [comment, marked as answer]
Whatever. I just hosted it on github pages and it works.
